.editorconfig

```diff
@@ -1,12 +1,15 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
+
 root = true
 
 [*]
-end_of_line = lf
-indent_style = space
-indent_space = 4
 charset = utf-8
-trim_trailing_whitespace = true
+indent_size = 4
```

Comment:
This was `indent_space`; please confirm which is correct.

Reply:
"Size" is correct. https://editorconfig-specification.readthedocs.io/#supported-pairs